projects/commudle-admin/src/app/feature-modules/public-community/components/home-community/home-community.component.scss

```diff
@@ -22,34 +29,53 @@
 
     .details {
       h1 {
-        margin-top: 0;
+        font-weight: $fw-600;
+        margin: $space-28 0 $space-8 0;
+        line-height: 2.4rem;
+        font-size: $fs-32;
       }
 
       .membership {
         display: flex;
-        align-items: center;
+        align-items: start;
         flex-direction: column;
+        .membership-toggle {
+          ::ng-deep {
+            button {
+              padding: $space-8;
+              font-weight: $fw-400;
+              background-color: #536EFF;
+              color: white;
+            }
+            .select-button {
+              padding: 4px;
+              background-color:initial;
+              color: initial;
+            }
+          }
+        }
 
         .member-count {
+          margin: 0;
+          margin-bottom: $space-78;
           a {
             text-decoration: none;
             color: black;
           }
         }
-
-        .membership-toggle {
-          margin: 0 0.8rem;
-        }
       }
 
       .community-social-links {
         display: flex;
```

Comment:
Let's try to use some the common classes defined for flex utilities directly in the HTML